### DISCUSSION TITLE
⚙️ Autogenerated Icons

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#1448] [Ideas]
Moving the discussion from #395 to here.

Summing it up so far, Arcticons itself isn't the place for auto-generated icons, but a companion app may be the solution. For example, the [Iconeration](https://codeberg.org/kaanelloed/Iconeration) app by @kaanelloed. I've started to link it in the request section of the app.

**Comment by npnpatidar** [comment]
How to use this, Is there an apk file to install this ?

**Reply by Donnnno** [reply]
You can download it here https://codeberg.org/kaanelloed/Iconeration/releases/tag/V0.6.0

**Reply by npnpatidar** [reply]
thanks :)
